javalib/src/main/scala/java/nio/file/attribute/FileTime.scala

```diff
@@ -3,37 +3,43 @@ package java.nio.file.attribute
 import java.util.concurrent.TimeUnit
 import java.time.Instant
 
-final class FileTime private (instant: Instant) extends Comparable[FileTime] {
-  def compareTo(other: FileTime) =
-    instant.compareTo(other.toInstant())
+final class FileTime private (private val epochMillis: Long,
+                              private val nanos: Int)
```

Comment:
If we store `epochMillis` and `nanos`, we will not be able to represent all the values that `FileTime` is supposed to hold. From looking at [the JavaDoc](https://docs.oracle.com/javase/8/docs/api/java/nio/file/attribute/FileTime.html), it's not clear, but I got the intuition that the canonical representation is a `value: Long` and `unit: TimeUnit`. Experimentation in a REPL suggests that my intuition is correct:
```scala
scala> import java.nio.file.attribute.FileTime
import java.nio.file.attribute.FileTime

scala> import java.util.concurrent.TimeUnit
import java.util.concurrent.TimeUnit

scala> FileTime.from(Long.MaxValue - 10L, TimeUnit.DAYS)
val res10: java.nio.file.attribute.FileTime = 219250468-12-04T15:30:07Z

scala> res10.to(TimeUnit.DAYS)
val res11: Long = 9223372036854775797 // exact number of days kept

scala> res10.to(TimeUnit.MILLISECONDS)
val res12: Long = 9223372036854775807 // overflowed to Long.MaxValue

scala> FileTime.from(Long.MaxValue - 10L, TimeUnit.NANOSECONDS)
val res13: java.nio.file.attribute.FileTime = 2262-04-11T23:47:16.854775797Z

scala> res13.to(TimeUnit.DAYS)
val res15: Long = 106751

scala> res13.to(TimeUnit.NANOSECONDS)
val res16: Long = 9223372036854775797 // exact number of nanoseconds kept
```
Therefore, I strongly believe that we should store `value: Long` and `unit: TimeUnit`, and adapt everything in terms of that canonical representation.

The above REPL session can be a good source of additional test cases.